stable/prometheus-mongodb-exporter/README.md

```diff
@@ -42,6 +42,7 @@ podAnnotations:
 | `image.tag` | MongoDB Exporter image tag | `0.7.0` |
 | `imagePullSecrets` | List of container registry secrets | `[]` |
 | `mongodb.uri` | The required [URI](https://docs.mongodb.com/manual/reference/connection-string) to connect to MongoDB | `` |
+| `secret.mongodbUri` | [URI](https://docs.mongodb.com/manual/reference/connection-string) to connect to MongoDB | `` |
```

Comment:
I don't like the idea of maintaining two ways to provide the MongoDB URI. Let's just keep `mongodb.uri` but expose it as env var `MONGODB_URI` only and remove the command line param. The only requirement is that people use v0.7.0, but they have to do that anyway since the args changed from `-` to `--`.

Reply:
Done ! I didn't change the Chart Version since I don't think this is a breaking change.